lib/client.js

```diff
@@ -132,6 +135,14 @@ Client.prototype.getJwtFromUrl = function () {
 };
 
 /**
+ * Attempts to replace the JWT in the url with the new JWT passed in the function.
+ * @param {string} newJwt
+ */
+Client.prototype.replaceJWTInUrl = function (newJwt) {
+  window.location.hash = '/?jwt=' + newJwt;
```

Comment:
I've fixed the test problem, and edited the error messages. Good to merge & `grunt release`? 

By the way, Contributing.md is super useful; I am not a frontend js guy so it helped me get the toolchain set up really easily =]